modules/Users/views/view.detail.php

```diff
@@ -122,6 +122,11 @@ function preDisplay() {
                     }
                 }
             }
+            if(is_admin($current_user)){
```

Comment:
yes but your inside that elseif check, so that is right I am talking about the if statement on line 111